cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -199,17 +200,47 @@ func generateMountsModeRun() error {
 	if err != nil {
 		return err
 	}
-	// 2.2 mount base
+
+	// 2.2.1 check if base is mounted
 	isBaseMounted, err := osutilIsMounted(filepath.Join(runMnt, "base"))
 	if err != nil {
 		return err
 	}
 	if !isBaseMounted {
-		base := filepath.Join(dataDir, "system-data", dirs.SnapBlobDir, modeEnv.Base)
-		fmt.Fprintf(stdout, "%s %s\n", base, filepath.Join(runMnt, "base"))
+		// 2.2.2 use modeenv base_status and try_base to see  if we are trying
+		// an update to the base snap
```

Comment:
we probably want the same kind of TODO comment about moving some of this to an helper in the boot package

Reply:
I will do this in #8077

Reply:
Added this in https://github.com/snapcore/snapd/pull/8077/commits/543a915f388691f867b98a2b43e58e630661165e